defaults_windows.go

```diff
@@ -52,7 +52,7 @@ func (m windowsmatcher) Match(p specs.Platform) bool {
 
 	if match && m.OS == "windows" {
 		// HPC containers do not have OS version filled
-		if p.OSVersion == "" {
+		if m.OSVersion == "" || p.OSVersion == "" {
```

Comment:
did we hit a case where the host OSVersion is returning "" ? I guess that should be a bug?

Reply:
Yes, in trying to match configurations. We may match the other direction, in this case we define a platform we want to use for unpacker and then we match against plugin platforms. By default plugin platforms are the default spec but can be overridden by user values such as for proxy plugins. We may be matching two user provided values though which are very unlikely to have the version.

Reply:
got it, thanks!